packages/cli/src/prompts.ts

```diff
@@ -155,6 +155,7 @@ The steps are: I. Fetching configs, II. Calculating difference and III. Applying
   public static readonly NO_CONFIGURED_SERVICES = 'There are not configured services in this environment'
   public static readonly NO_ADDITIONAL_CONFIGURED_SERVICES = 'There are no additional configurable services for this environment'
   public static readonly SERVICE_ALREADY_ADDED = (serviceName: string): string => `${serviceName} was already added to this environment`
+  public static readonly SERVICE_NOT_VALID = (serviceName: string, supportedServiceAdapters:string[]): string => `${serviceName} is not a valid service name, available service names are:\n${supportedServiceAdapters.join('\n')}`
```

Comment:
I think a more proper name here is SERVICE_NAME_NOT_VALID. Better to be more particular
```suggestion
  public static readonly SERVICE_NAME_NOT_VALID = (serviceName: string, supportedServiceAdapters: string[]): string => `${serviceName} is not a valid service name, available service names are:\n${supportedServiceAdapters.join('\n')}`
```